theme/js/script.js

```diff
@@ -44,46 +44,53 @@ $(document).ready(function() {
 
     countCharacter();
 
+    openDonationContent();
+
     /* Calling the clickableElement function on card-wrapper.
     This function needs parameters running through it to work. */
     clickableElement($('.card-component-container .card-wrapper'));
 
+    document.getElementById("defaultOpen").click();
```

Comment:
You don’t need this line, you can remove it.

Reply:
done